tests/components/harmony/test_config_flow.py

```diff
@@ -91,19 +91,28 @@ def _get_mock_harmonyapi(connect=None, close=None):
     """Test we get the form with ssdp source."""
     await setup.async_setup_component(hass, "persistent_notification", {})
 
-    result = await hass.config_entries.flow.async_init(
-        DOMAIN,
-        context={"source": config_entries.SOURCE_SSDP},
-        data={
-            "friendlyName": "Harmony Hub",
-            "ssdp_location": "http://192.168.209.238:8088/description",
-        },
-    )
+    harmonyapi = _get_mock_harmonyapi(connect=True)
+
+    with patch(
+        "homeassistant.components.harmony.config_flow.HarmonyAPI",
+        return_value=harmonyapi,
+    ):
+        result = await hass.config_entries.flow.async_init(
+            DOMAIN,
+            context={"source": config_entries.SOURCE_SSDP},
+            data={
+                "friendlyName": "Harmony Hub",
+                "ssdp_location": "http://192.168.1.12:8088/description",
+            },
+        )
     assert result["type"] == "form"
     assert result["step_id"] == "link"
     assert result["errors"] == {}
+    assert result["description_placeholders"] == {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5224791)